asyncapi/json-schemas/operational/assignment/attempt/request/common/assignment-attempt-journey-call.json

```diff
@@ -0,0 +1,28 @@
+{
+  "$id": "https://schemas.ruter.no/adt/ota/api/v3.0/operational/assignment/attempt/request/common/assignment-attempt-journey-call.json",
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "title": "AssignmentAttemptJourneyCall",
+  "description": "Assignment journey call used defining calls in a journey",
```

Comment:
could something like "An Assignment journey call defines the specific stops (calls) that a vehicle will make during a journey." be more easier to understand?